requirements/app/app.txt

```diff
@@ -1,4 +1,4 @@
-lightning-cloud ==0.5.39  # Must be pinned to ensure compatibility
+lightning-cloud ==0.5.41  # Must be pinned to ensure compatibility
```

Comment:
@ethanwharris made this PR https://github.com/Lightning-AI/lightning/pull/18778 to pin to 39, could you check with him please

Reply:
This should be fine now. Ethan added back the missing API.

Reply:
Yeah, this is fine, main thing is that we pin to a single version and don't allow a range. Thanks for checking @awaelchli !